### ISSUE TITLE
PANIC:Invalid `prisma.user.findOne()` invocation 

### ISSUE THREAD

**Issue by lmichratsimbazafy** [prisma/prisma-client-js#870]
Hi Prisma Team! My Prisma Client just crashed. This is the report:
## Versions

| Name     | Version            |
|----------|--------------------|
| Node     | v12.18.3           | 
| OS       | debian-openssl-1.1.x|
| Prisma   | 2.5.1              |

## Logs
```
prisma-client {
  engineConfig: {
    cwd: '/server/prisma',
    enableDebugLogs: false,
    enableEngineDebugMode: undefined,
    datamodelPath: '/server/node_modules/.prisma/client/schema.prisma',
    prismaPath: undefined,
    engineEndpoint: undefined,
    generator: {
      name: 'client',
      provider: 'prisma-client-js',
      output: '/server/node_modules/@prisma/client',
      binaryTargets: [],
      previewFeatures: [],
      config: {}
    },
    showColors: false,
    logLevel: undefined,
    logQueries: undefined,
    flags: [],
    clientVersion: '2.5.1',
    enableExperimental: [],
    useUds: undefined
  }
}
plusX Execution permissions of /server/node_modules/.prisma/client/query-engine-debian-openssl-1.1.x are fine
prisma-client Client Version 2.5.1
prisma-client Engine Version 6a8054bb549e4cc23f157b0010cb2e95cb2637fb
```

**Comment by lmichratsimbazafy** [comment]
Fixed by downgrading @prisma/client and @prisma/cli version to 2.5.1

**Comment by janpio** [comment]
Your report shows you already having Prisma Client 2.5.1. What steps exactly did you do to fix this?

**Comment by lmichratsimbazafy** [comment]
2.5.1 is CLIENT version when I tried to downgrade it from 2.6.1  but CLI version was still 2.6.1 so I had to downgrade both two with the same version (2.5.1)

**Comment by janpio** [comment]
Ok, so at the time of report you were using CLI 2.6.1 and Client 2.5.1 - correct?
Can you share the query you executed so we can possibly reproduce this? We are very interested in understanding this version mismatches and what is happening there.

**Comment by lmichratsimbazafy** [comment]
yeah, right, but at the beginning of this bug, I used both Client and CLI 2.6.1 so I tried to downgrade only the Client but I still had the same error. This my query      
`const user = await ctx.prisma.user.findMany({
      where: {
        roles: { some: { name:'ADMIN' } },
        OR: [
          { email: { contains: filter } },
          { lastName: { contains: filter } },
          { firstName: { contains: filter } },
        ],
      },
    });  `

here is the graphQl client query

`  query GetAdmins($filter: String) {
    getAdmins(input: {filter: $filter}) {
    id
    email
    firstName
    lastName
    roles
    }
  }
` 
when I remove the 'roles' field from the client query, this works fine.
Here is the roles field resolver:
`
  roles: async (user, {id}: , ctx): => {
  .........
  return await   ctx.prisma.user.findOne(
    {
      where: { id },
      include: { roles: true },
    },
  );

  },
`

**Comment by pantharshit00** [comment]
@lmichratsimbazafy I am sorry for the late reply here. Can you reproduce this in the latest version as well?
 
 If yes, please share your schema as well so that we can try a reproduction.
